Sources/Xcodeproj/pbxproj().swift

```diff
@@ -413,14 +413,14 @@ func xcodeProject(
 
         if target.type == .test {
             targetSettings.common.EMBEDDED_CONTENT_CONTAINS_SWIFT = "YES"
-            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["@loader_path/../Frameworks", "@loader_path/Frameworks"]
+            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["@loader_path/../Frameworks", "@loader_path/Frameworks", "$(inherited)"]
         } else {
             // We currently force a search path to the toolchain, since we can't
             // establish an expected location for the Swift standard libraries.
             //
             // Note that this means that the built binaries are not suitable for
             // distribution, among other things.
-            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["$(TOOLCHAIN_DIR)/usr/lib/swift/macosx"]
+            targetSettings.common.LD_RUNPATH_SEARCH_PATHS = ["$(TOOLCHAIN_DIR)/usr/lib/swift/macosx", "$(inherited)"]
```

Comment:
Isn't it more common for `$(inherited)` to be the first element? If not, this looks fine.
/cc @abertelrud